extensions/wikia/PortableInfobox/js/PortableInfoboxBuilder.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+require(['wikia.querystring', 'jquery', 'wikia.ui.factory'], function (queryString, $, uiFactory) {
+	var initBuilderUIqueryStringParam = 'portableInfoboxBuilder',
+		initBuilderUIEvantName = 'initPortableInfoboxBuilder',
+		builderUIModalEntryPointId = 'portableInfoboxBuilderModalEntryPoint',
+		builderUIModalEntryPointParams = {
+			vars: {
+				id: builderUIModalEntryPointId,
+				size: 'small',
```

Comment:
this is required to set a modal size. 3 different sizes are supported `small`, `medium` and `large`. This is later use to set a class name to modal component which applies different styles for different sizes